pkg/kubelet/eviction/helpers_test.go

```diff
@@ -110,6 +110,22 @@ func TestGetReclaimableThreshold(t *testing.T) {
 			t.Errorf("Didn't find reclaimable threshold, test: %v", testName)
 		}
 	}
+	cornerCases := map[string]struct {
```

Comment:
Can this case be added to the `testCases` instead?

Reply:
resolved in e732e7535ae